recipes/jupyterlab-sos/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyterlab-sos" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://registry.npmjs.org/jupyterlab-sos/-/jupyterlab-sos-{{ version }}.tgz
+  sha256: afbfa0c906a45db6e97b12731d308de89e80990833401355198f75f455328de5
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I don't think this is noarch python if there are separate build scripts

Reply:
Ohmm, I agree. Will change soon.

Reply:
It should be noarch, as the package does not contain any compiled or architecture specific code... The fact that there are separate build scripts is just for the case that somebody wants to build the noarch package on windows... other jupyterlab packages are build the same way and are also noarch.

Reply:
Changed back to `noarch`.